### ISSUE TITLE
Logging: Missing or incomplete language bundles generates far too much logging, could maybe be more helpful or improve doc.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#8446]
In the process of testing various language, custom metadata blocks, metadata languages, and custom controlled vocabulary, I've accumulated some non standard bits around metadata. Although I try to cleanly configure/unconfigure language, I'm still seeing numerous complaints in the logs about missing bundles and they are quite noisy, posting pages upon pages of the repeated warnings about not finding the bundles.

While I don't want to ask to over engineer to correct what some may consider a test only environment, it seems there is something here that doesn't quite jibe and so needs to be teased apart. This issue is a place holder while I do that.

**Comment by kcondon** [comment]
What I've found so far in researching my test env's issues:
1. Syntax errors in custom metadata bloc properties files, customMRA, customGSD, using the char ':' as a separator, not as an '=' as first thought. A separate ticket is open for this 
2. Custom metadata blocks (now) require an accompanying <block>.properties file always, even the default case where no language is defined, see https://github.com/IQSS/dataverse/issues/4684 and https://github.com/IQSS/dataverse/pull/5290/commits/8dd259346bbee69abf0f07c34223edfb9eff29cd So, this was a user error.
3. When enabling both en and fr, using the dataverse-v5.9 branch, I noticed many _en and _fr bundle issues due to the custom blocks but also some missing _zh errors, all when loading the homepage. It seems we log too many of these errors to be entirely useful IMHO and need to track down the source of the _zh errors.
